src/securejoin.rs

```diff
@@ -1444,16 +1458,25 @@ mod tests {
             "vg-member-added-received"
         );
 
-        let bob_chatid = joiner.await;
         let bob_chat = Chat::load_from_db(&bob.ctx, bob_chatid).await?;
         assert!(bob_chat.is_protected());
+        assert!(bob_chat.typ == Chattype::Group);
         assert!(!bob.ctx.has_ongoing().await);
 
         // On this "happy path", Alice and Bob get only a group-chat where all information are added to.
         // The one-to-one chats are used internally for the hidden handshake messages,
         // however, should not be visible in the UIs.
         assert_eq!(Chatlist::try_load(&alice, 0, None, None).await?.len(), 1);
         assert_eq!(Chatlist::try_load(&bob, 0, None, None).await?.len(), 1);
+
+        // If Bob then sends a direct message to alice, however, the one-to-one with Alice should appear.
+        let bobs_chat_with_alice = bob.create_chat(&alice).await;
+        send_text_msg(&bob, bobs_chat_with_alice.id, "Hello".to_string()).await?;
+        let sent = bob.pop_sent_msg().await;
```

Comment:
There is a shortcut `let sent = bob.send_text(bobs_chat_with_alice.id, "Hello").await` in `test_utils.rs`.

Reply:
i changed that, thanks for the hint!